questionnaire-zh-zh.json

```diff
@@ -0,0 +1,434 @@
+{
+  "specVersion": "2.1.1",
+  "language": "zh-zh",
+  "title": "合规性问题一览",
+  "contextText": "背景",
+  "questionsText": "问题",
+  "completedText": "已完成？",
+  "statusText": "状态",
+  "answerColumnText": "回答",
+  "sectionColumnText": "部分",
+  "numberColumnText": "编号",
+  "specRefColumnText": "规范参考",
+  "questionColumnText": "问题文本",
+  "tocText": "目录",
+  "preambleText": "OpenChain项目通过简化开放源许可合规性和保持一致性，建立对于开放源的信任。维护OpenChain规范，定义各品质合规程序必须满足的一组核心需求。其结果是，对于软件供应链参与者来说，开放源许可合规性变得更加可预测、可理解和高效。\n本文档包含一系列确定公司是否符合OpenChain的问题。如果所有问题的答案都为“是”，那么该公司符合OpenChain规范2.1版（功能与ISO\/IEC DIS 5230:2020（e）一致）的所有要求。如果任一问题的答案为“否”，那么公司可以清楚地确定哪些需要追加投资以改进合规程序。",

```

Comment:
Not sure about the backslash in the preamble text

Reply:
done locally, PR later.